### ISSUE TITLE
Town Guard attacked by Iron Golems

### ISSUE THREAD

**Issue by lartts** [Lothrazar/VillagerTools#6]
**Minecraft Version:** 
1.16.5

**Forge Version:** 
36.2.2

**Mod Version:** 
1.0.1

**Single Player or Server:**
Single Player

**Describe problem (what you were doing; what happened; what should have happened):**
Testing out town guards, spawning them alongside iron golems, villager guards (Guard Villagers), and gatekeepers (Blue Skies). They all attack the town guards, I'm pretty sure town guards are supposed to defend villages so it wouldn't make sense for other village defenders to attack town guards.

**Log file link:**
Doesn't give information

**Video/images/gifs (direct upload or link):** 
![2021-08-01_13 50 00](https://user-images.githubusercontent.com/74058566/127762119-de22686f-57c5-4ec2-9fbf-2fedf17dfe20.png)
![2021-08-01_13 51 05](https://user-images.githubusercontent.com/74058566/127762123-477c9cb3-fc13-42c7-9c68-d5ff7c1fd525.png)
![2021-08-01_13 51 25](https://user-images.githubusercontent.com/74058566/127762125-11566152-4a07-4883-b2b2-0072157a5066.png)

**Comment by huyhhuy1410** [comment]
Yeah, i have same issue. This issue made Town Guards seem useless
